SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are only provided for last released version.
+Previous versions are not fixed.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.8.x   | :white_check_mark: |
+| < 0.8.0 | :x:                |
+
+## Reporting a Vulnerability
+
+If any can prevent avahi-daemon from responding or even terminating it
+Use this section to tell people how to report a vulnerability.
+If you discover a way to crash avahi-daemon via network packet or make it non-responsive,
+please do not create normal public issue. Instead report it on
+[Security](https://github.com/avahi/avahi/security) page as a vulnerability.
```

Comment:
There should probably be a way to report things without having a GitHub account.